aerospike-core/src/client.rs

```diff
@@ -54,7 +54,6 @@ use crate::{
 /// relevant subset of bins.
 pub struct Client {
     cluster: Arc<Cluster>,
-    thread_pool: Pool,
 }
 
 unsafe impl Send for Client {}
```

Comment:
It is a convention to have a safety comment above `unsafe`, explaining the invariant, why it is okay.